Crash due to IllegalStateException on badge access in MainActivity

Had the following crash happen while working on an unrelated project:

```
 AndroidRuntime  D  Shutting down VM
                 E  FATAL EXCEPTION: main
                 E  Process: com.woocommerce.android, PID: 16599
                 E  java.lang.RuntimeException: Unable to resume activity {com.woocommerce.android/com.woocommerce.android.ui.main.MainActivity}: java.lang.IllegalStateException: badge must not be null
                 E      at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3784)
                 E      at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3816)
                 E      at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:51)
                 E      at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:145)
                 E      at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70)
                 E      at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1808)
                 E      at android.os.Handler.dispatchMessage(Handler.java:106)
                 E      at android.os.Looper.loop(Looper.java:193)
                 E      at android.app.ActivityThread.main(ActivityThread.java:6669)
                 E      at java.lang.reflect.Method.invoke(Native Method)
                 E      at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
                 E      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
                 E  Caused by: java.lang.IllegalStateException: badge must not be null
                 E      at com.woocommerce.android.ui.main.MainActivity.showNotificationBadge(MainActivity.kt:289)
                 E      at com.woocommerce.android.ui.main.MainActivity.updateNotificationBadge(MainActivity.kt:283)
                 E      at com.woocommerce.android.ui.main.MainActivity.onResume(MainActivity.kt:136)
                 E      at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1412)
                 E      at android.app.Activity.performResume(Activity.java:7292)
                 E      at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3776)
                 E      ... 11 more
        Process  I  Sending signal. PID: 16599 SIG: 9
```

--- a/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/main/MainActivity.kt
+++ b/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/main/MainActivity.kt
@@ -284,10 +284,12 @@ class MainActivity : AppCompatActivity(),
     }
 
     override fun showNotificationBadge(show: Boolean) {
-        if (show && badge.visibility != View.VISIBLE) {
-            WooAnimUtils.fadeIn(badge, Duration.MEDIUM)
-        } else if (!show && badge.visibility == View.VISIBLE) {
-            WooAnimUtils.fadeOut(badge, Duration.MEDIUM)
+        badge?.let {
+            if (show && it.visibility != View.VISIBLE) {
+                WooAnimUtils.fadeIn(it, Duration.MEDIUM)
+            } else if (!show && it.visibility == View.VISIBLE) {
+                WooAnimUtils.fadeOut(it, Duration.MEDIUM)
+            }
         }
     }
 

